### ISSUE TITLE
Transcribe 2 edge cases that come out of the new Twilio transcription service

### ISSUE THREAD

**Issue by daguar** [codeforamerica/balance#348]
There are two somewhat uncommon edge cases of transcription text found via QA that our new DollarAmountsProcessor ( done to fix #343 ) doesn't currently handle:

```
expected: "They were snapped balances. $10.22. Your cash balance is $1.00 to repeat your account balance. Press one. To hear your last ten transactions on your card. Press two to change European press three. To report."

got: "They were snapped balances. Ten dollars and twenty two cents. Your cash balance is $1.00 to repeat your account balance. Press one. To hear your last ten transactions on your card. Press two to change European press three. To report."
```

```
expected: "Nap balances. $649.00. Your cash balance is $0.00. As a reminder by saving the receipt from your last purchase you will know your current balance. Remember you can also access your account information on."

got: "Nap balances. Six $149.00. Your cash balance is $0.00. As a reminder by saving the receipt from your last purchase you will know your current balance. Remember you can also access your account information on."
```

I've added (failing) tests for these edge cases on a new branch in case someone has some time and wants to take a stab at modifying the DollarAmountsProcessor to get them: [new-transcription-service-edge-cases-010716](https://github.com/codeforamerica/balance/tree/new-transcription-service-edge-cases-010716)

**Comment by etrepum** [comment]
Two char fix in https://github.com/codeforamerica/balance/pull/349

**Comment by daguar** [comment]
Closed by #349 — thanks again @etrepum !
